### ISSUE TITLE
NTR: MRI: putamen hyperintensity (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8893]
New term request MRI: putamen hyperintensity (kalia_091015015653):

**Comment by pnrobinson** [comment]
@MickeySegal
is this  hyperintense putaminal rim (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7051745/)?

**Comment by MickeySegal** [comment]
Yes.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001084
- label: T2-hyperintense putaminal rim
- parents: [Abnormal putamen morphology (HP:0031982)](https://hpo.jax.org/app/browse/term/HP:0031982)
- pmid: PMID:19147717
- def: Increased signal on T2 magnetic resonance imaging characterized by a linear region of high signal surrounding the lateral aspect of the putamen at 1.5 Tesla.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8893
- synonyms:
